client/app/components/OnboardingTopBar/OnboardingTopBar.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react';
+
+import css from './OnboardingTopBar.scss';
+
+class OnboardingTopBar extends React.Component {
```

Comment:
Given the fact that there isn't anything stateful in this component (which is a good thing), there are several methods that take their input from `this.props`. Could those methods be refactored as separate functions, probably within this same file (if not really useful elsewhere)?

Having several clearly pure functions would IMO help a lot when reading through what thas component does and what it depends on. This would also make it easier to port the components as stateless function components in the future if we decide so.

I know the call site of the functions/methods will have more characters, but I'd say being explicit helps in reasoning about the code (sorry for the cliché saying).